auspice/server/getAvailable.js

```diff
@@ -7,7 +7,7 @@ const getAvailable = async (req, res) => {
   const prefix = queryString.parse(req.url.split('?')[1]).prefix || "";
   utils.verbose(`getAvailable prefix: "${prefix}"`);
 
-  const {source} = splitPrefixIntoParts(prefix);
+  const {source, isGroup} = splitPrefixIntoParts(prefix);
```

Comment:
Instead of threading through a boolean (as another variable that needs preservation and passing around), I think `splitPrefixIntoParts` and `joinPartsIntoPrefix` can do what they need via introspection of the `source` instance.